images/ansible-operator/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN set -e && yum clean all && rm -rf /var/cache/yum/* \
   && pip3 install --upgrade pip~=23.3.2 \
   && pip3 install pipenv==2023.11.15 \
   && pipenv install --deploy \
-  && pipenv check \
+  && pipenv check --ignore 70612 \
```

Comment:
What is this CVE and why are we ignoring it?

Reply:
This was a very old CVE that just got re-triggered, and is disputed in a few places.

https://github.com/advisories/GHSA-f6pv-j8mr-w6rr
https://github.com/dbt-labs/dbt-core/issues/10250
https://data.safetycli.com/v/70612/97c/